content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -739,6 +584,17 @@ query.iter().for_each(|mut component| {
 
 These abstractions were introduced in [#6404](https://github.com/bevyengine/bevy/pull/6404), [#7381](https://github.com/bevyengine/bevy/pull/7381) and [#7568](https://github.com/bevyengine/bevy/pull/7568).
 
+## Cubic Curves
+
+<div class="release-feature-authors">authors: @aevyrie</div>
+
+In preparation for UI animation and hand-tweaked animation curves cubic curves have been added to `bevy_math` with a public `CubicGenerator` trait to enable user-defined cubic curves. The implementation provides multiple curves out of the box, useful in various applications:
```

Comment:
```suggestion
In preparation for UI animation and hand-tweaked animation curves, cubic curves have been added to `bevy_math`.  The implementation provides multiple curves out of the box, useful in various applications:
```

Then below discuss the  "public `CubicGenerator` trait to enable user-defined cubic curves".